packages/hash/api/src/model/ontology/data-type.model.ts

```diff
@@ -53,13 +52,17 @@ export default class {
    */
   static async getAllLatest(
     graphApi: GraphApi,
-    params: { accountId: string },
+    _params: { accountId: string },
   ): Promise<DataTypeModel[]> {
     /** @todo: get all latest data types in specified account */
-    const { data: schemas } = await graphApi.getLatestDataTypes();
+    const { data: identifiedDataTypes } = await graphApi.getLatestDataTypes();
```

Comment:
Perhaps let's just refer to the response as `dataTypes` here? I'm not sure `identifiedDataTypes ` makes a lot of sense in this context.
```suggestion
    const { data: dataTypes } = await graphApi.getLatestDataTypes();
```

Reply:
I was verbose on purpose as a `DataType` would refer to the instance of the JSON Schema. I think it could become quite an overloaded term if we refer to it as `DataType` in many places

Reply:
Did you intend on using `identifiableDataTypes` instead? `identifiedDataTypes` doesn't convey any more meaning than `dataTypes` in my mind, but perhaps I'm wrong.

Regardless this doesn't block merging, just wanted to point out that I'm not sure whether the naming is ideal here.

Reply:
Calling them `persistedDataType` instead as that's the name of the spec type, is that okay? :)